com.hack_computer.Clubhouse.json

```
@@ -0,0 +1,276 @@
+            "ensure-writable": [
+                "easy-install.pth"
+            ],
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} python-dbusmock"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/d3/61/dcc36e58577446c70ffa1cdca4c93dae5fd651c840eaf5717d98a4560aa7/python-dbusmock-0.18.3.tar.gz",
+                    "sha256": "994a178268b6d74aeb158c0f155cd141e9a0cfae14226a764cd022c4949fe242"
+                }
+            ]
+        },
+        {
+            "name": "eos-metrics",
+            "config-opts": [
+                "--enable-gir-doc=no"
+            ],
+            "build-options": {
+                "env": {
+                    "PYTHONPATH": "/app/lib/python3.7/site-packages/"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/endlessm/eos-metrics.git",
+                    "tag": "Release_3.8.6",
+                    "commit": "6b78996c7aa86892227c2ab6cbb8c173f469e4ec"
+                }
+            ]
+        },
+
+        {
+          "name": "Metropolis-font",
+          "buildsystem": "simple",
+          "build-commands": [
+            "install -D -m644 *.otf -t /app/share/fonts"
+          ],
+          "post-install": [
+            "fc-cache -f /app/share/fonts/"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/chrismsimpson/Metropolis/archive/r9.tar.gz",
+              "sha256": "283d1f51d96777c648dc1c0899b986fc99f14bfb709732428c14881c075c356b"
+            }
+          ]
+        },
+        {
+          "name": "Lato-font",
+          "buildsystem": "simple",
+          "build-commands": [
+            "install -D -m644 *.ttf -t /app/share/fonts"
+          ],
+          "post-install": [
+            "fc-cache -f /app/share/fonts/"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "http://www.latofonts.com/download/Lato2OFL.zip",
+              "sha256": "42b54e96c07e299d967fc3227c7bd63a20d6cfb1dc8fd6dae83628091e20a5b8"
+            }
+          ]
+        },
+        {
+          "name": "Hack-font",
+          "buildsystem": "simple",
+          "build-commands": [
+            "install -D -m644 *.ttf -t /app/share/fonts"
+          ],
+          "post-install": [
+            "fc-cache -f /app/share/fonts/"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/source-foundry/Hack/releases/download/v3.003/Hack-v3.003-ttf.tar.xz",
+              "sha256": "d9ed5d0a07525c7e7bd587b4364e4bc41021dd668658d09864453d9bb374a78d"
+            }
+          ]
+        },
```

```suggestion        {
          "name": "fonts",
          "buildsystem": "simple",
          "build-commands": [
            "install -D -m644 *.otf *.ttf -t /app/share/fonts"
          ],
          "post-install": [
            "fc-cache -f /app/share/fonts/"
          ],
          "sources": [
            {
              "type": "archive",
              "url": "https://github.com/chrismsimpson/Metropolis/archive/r9.tar.gz",
              "sha256": "283d1f51d96777c648dc1c0899b986fc99f14bfb709732428c14881c075c356b"
            },
            {
                "type": "archive",
                "url": "http://www.latofonts.com/download/Lato2OFL.zip",
                "sha256": "42b54e96c07e299d967fc3227c7bd63a20d6cfb1dc8fd6dae83628091e20a5b8"
            },
            {
                "type": "archive",
                "url": "https://github.com/source-foundry/Hack/releases/download/v3.003/Hack-v3.003-ttf.tar.xz",
                "sha256": "d9ed5d0a07525c7e7bd587b4364e4bc41021dd668658d09864453d9bb374a78d"
            }
          ]
        },
        
```

You could simplify this a lot